.gitignore

```diff
@@ -3,4 +3,5 @@ bower_components
 node_modules
 /vg-tooltip.min.js
 /vg-tooltip.css
+src/vg-tooltip.js
```

Comment:
Actually you shouldn't add this to both npm and gitignore, but should simply remove it from the src folder.  (`git rm src/vg-tooltip.js`)